packages/plugins/package.json

```diff
@@ -68,6 +67,8 @@
     "hardhat": "2.20.1"
   },
   "devDependencies": {
+    "sphinx-forge-std": "https://github.com/foundry-rs/forge-std.git#v1.7.1",
+    "sphinx-solmate": "npm:solmate@^6.7.0",
```

Comment:
I just noticed Solmate doesn't need to be a prod dependency, so I moved it over, too.